android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -259,12 +259,9 @@ public static void openBuySendSwapActivity(Activity activity,
             BuySendSwapActivity.ActivityType activityType, String swapFromAssetSymbol,
             String chainId) {
         assert activity != null;
-        Intent buySendSwapActivityIntent = new Intent(activity, BuySendSwapActivity.class);
-        buySendSwapActivityIntent.putExtra(
-                BuySendSwapActivity.ACTIVITY_TYPE, activityType.getValue());
-        buySendSwapActivityIntent.putExtra(BuySendSwapActivity.ASSET_SYMBOL, swapFromAssetSymbol);
-        buySendSwapActivityIntent.putExtra(BuySendSwapActivity.ASSET_CHAIN_ID, chainId);
-        activity.startActivity(buySendSwapActivityIntent);
+
+        BraveActivity.getBraveActivity().openNewOrSelectExistingTab(BraveActivity.BRAVE_SWAP_URL);
```

Comment:
we should do that only for `activityType == BuySendSwapActivity.ActivityType.SWAP`. Leave the old `Intent` path for other types as we want to use it for Buy and Send

Reply:
Fixed